### ISSUE TITLE
Synchronise layout - for when you want to go round the room doing intros, and you don't want it to be awkward.

### ISSUE THREAD

**Issue by lampholder** [element-hq/element-call#238]
**Is your feature request related to a problem? Please describe.**
It's always super awkward when folks are trying to intro themselves on a VC but everybody has a random layout so you can't just go "round the room" like you would in real life.

**Describe the solution you'd like**
Either a moderator can impose or a participant can propose that everybody share their screen layout.

**Describe alternatives you've considered**
We all stop doing video calling because it's too awkward.

**Additional context**
Nada

**Comment by ara4n** [comment]
+1000000

**Comment by jakewb-b** [comment]
I agree, though this does create some complexity with the whole idea of having the freedom layout that is entirely personal to me. 

A couple of ideas spring to mind:
- An admin could lock freedom layout, meaning that all users see the admin's freedom layout. 
- There could be a third layout that is not freedom, but is consistent for everybody. Although, it doesn't add a lot of value except for the 'going round the room' scenario. 

@gaelledel - something for you to have a think about for future iterations, I think.

**Comment by fkwp** [comment]
In the meantime the UX got some iterations. Not sure if that issue is still present. Closing it right now since it is quite stale for some time. Please reopen if that is still an issue
